crates/bevy_ecs/src/event.rs

```diff
@@ -248,7 +248,48 @@ impl<'w, 's, E: Event> EventReader<'w, 's, E> {
     }
 }
 
-/// Sends events of type `T`.
+/// Sends events of type `T` to [`EventReader`]s of type `T`.
```

Comment:
I know this makes sense from a high level perspective, but technically the events are sent to `Events<T>` and `EventReader` is just a nicer abstraction around that.

Personally I would simply not mention it at all

```suggestion
/// Sends events of type `T`.
```